cincinnati/src/lib.rs

```diff
@@ -154,25 +151,27 @@ pub struct Empty;
 
 /// Errors that can be returned by the methods in this library
 pub mod errors {
+    use commons::prelude_errors::*;
```

Comment:
I have not seen this before. Can you please explain purpose of this?

Reply:
Also did not find any pointers in internet e.g. https://doc.rust-lang.org/std/prelude/index.html

Reply:
`prelude` is a naming convention for modules which export commonly used items. AFAIU there's nothing right or wrong about it, and I decided to include `errors` in the name as this prelude is about error related items only.